includes/shared-assets/js/analytics.js

```diff
@@ -21,15 +21,43 @@ const maybeInitializeAnalytics = () => {
 	}
 };
 
+/**
+ * Checks to see if we should send GA event or not for specific conditions
+ * @returns bool
+ */
+const isExcluded = () => {
+	// exclude ga event or page view if domain is wpengine.com
+	const excludeRegex = /.+\.wpengine\.com/gi;
+	return excludeRegex.test(window.location.href);
+};
+
+/**
+ * Checks for tracking setting and exclusions
+ * @returns bool
+ */
+const shouldTrack = () => {
+	return isExcluded() && telemetryEnabled;
```

Comment:
Should this be _not_ excluded?

```suggestion
	return !isExcluded() && telemetryEnabled;
```